### DISCUSSION TITLE
the download does not work and error 403 (Forbidden) appears.

### DISCUSSION THREAD

**Discussion by helloidontknoww** [shaked6540/YoutubePlaylistDownloader#367] [Q&A]
I need help with this program actually
when I paste the link and click download, it loads and then shows me:

Here is a list of the videos that could not be downloaded, and the reason why: 
Smells Like Teen Spirit Reason: Response status code does not indicate success: 403 (Forbidden).
In Bloom Reason: Response status code does not indicate success: 403 (Forbidden).
Come As You Are Reason: Response status code does not indicate success: 403 (Forbidden).
Breed Reason: Response status code does not indicate success: 403 (Forbidden).
Lithium Reason: Response status code does not indicate success: 403 (Forbidden).
Polly Reason: Response status code does not indicate success: 403 (Forbidden).
Territorial Pissings Reason: Response status code does not indicate success: 403 (Forbidden).
Drain You Reason: Response status code does not indicate success: 403 (Forbidden).
Lounge Act Reason: Response status code does not indicate success: 403 (Forbidden).
Stay Away Reason: Response status code does not indicate success: 403 (Forbidden).
On A Plain Reason: Response status code does not indicate success: 403 (Forbidden).
Something In The Way Reason: Response status code does not indicate success: 403 (Forbidden).
Endless, Nameless Reason: Response status code does not indicate success: 403 (Forbidden).

and I literally see myself in every video, please answer quickly
why am I being shown this because I don't know why, or better yet, how
it was possible to update
![screen 1](https://github.com/user-attachments/assets/734beea7-79ea-4cac-9ae3-c4721fdd71ec)
![screen 2](https://github.com/user-attachments/assets/16aabbe9-81ab-4add-b9a9-2d99d3622eb2)

**Comment by cristianst85** [comment]
There are already about ten issues opened about this and one other discussion. There is also an open PR that should fix it.

https://github.com/shaked6540/YoutubePlaylistDownloader/pull/366

**Reply by helloidontknoww** [reply]
what PR, and why did you give the thumbs down, something doesn't suit you?

**Reply by helloidontknoww** [reply]
yes, yes I know there is a second article, but I need to highlight these mistakes

**Reply by cristianst85** [reply]
Thumbs down because you should not open an issue or discussion if there is already one that addresses the same problem you have. Otherwise, you are just spamming. The PR that I opened is to fix this issue.

**Reply by helloidontknoww** [reply]
I didn't want spam, I just have the same problem

**Reply by Empty23** [reply]
how to solve this error sometimes it worked but moslty it's like this error
